packages/compass-home/src/components/home.tsx

```diff
@@ -399,7 +399,7 @@ function ThemedHome(
   }, [setIsSettingsOpen]);
 
   return (
-    <LeafyGreenProvider>
+    <LeafyGreenProvider darkMode={theme.enabled && theme.theme === Theme.Dark}>
       <ThemeProvider theme={theme}>
```

Comment:
Why are we keeping our own provider? In what cases we should use leafygreen useDarkMode hook and in what cases ours?

Reply:
Updated and removed ours, I don't think there's any case currently where we'd need our own. Thanks for raising!